### PR TITLE
Syncing with changes made to docs

### PR DESCRIPTION
**Pull request by moxiegirl**
The `docs.blockstack` repo and the `docs` directory in this repo are out-of-sync.  This brings them back into sync.  The documentation on the `docs.blockstack` side is current with this.

Signed-off-by: Mary Anthony <mary@blockstack.com>

### CODE REVIEW

**Comment by kantai** [comment]
I think this should probably be merged to `develop` rather than `master` -- `master` doesn't have Clarity in it yet.

**Comment by moxiegirl** [comment]
@kantai The docs on master should reflect what is published in our public documentation.  The fact that clarity _code_ is in the `develop` branch is documented in our public docs.

**Comment by kantai** [comment]
Okay, that works for me

File: docs/api-specs.md [hunk 1]
```diff
@@ -2065,7 +2065,7 @@ If the account does not exist, then an empty list will be returned.
 # Group Resolver Endpoints
 
 ## Lookup User [GET /v1/users/{username}]
-Lookup and resolve a user's profile. Defaults to the `id` namespace.
+Lookup and resolver a user's profile. Defaults to the `id` namespace.
```
Comment (jcnelson) [inline docs/api-specs.md]: Typo -- should be `Look up and resolve`.

**Pull request merged**